run-tox-env.yml

```diff
@@ -62,7 +69,7 @@ jobs:
           fetchDepth: 999999999
 
         - ${{ if eq(env_pair.key, 'linux32') }}:
-          - bash: docker run -v $PWD:/project -i --name manylinux2010 -d quay.io/pypa/manylinux2010_i686:latest /bin/bash
```

Comment:
I think the manylinux should be configurable. Some packages aren't built for newer ones and then you have to compile them from source which leads to long build times and issues we see on SunPy's builds. (bumping the versions further probably makes this even worse)

Reply:
Sigh.